rtapp_function/Makefile

```diff
@@ -0,0 +1,14 @@
+SOURCE = $(wildcard */*.c)
```

Comment:
Mmm, I'm wondering if pulling "random" code into rt-app sources is really a good idea.
I guess we are basically subscribing to maintaining all the snippets we import.. :-(
Alternatives?

Reply:
Agree, this should be mostly a mechanism to allow for random code to be used as rt-app phases. Only very select and generic code snippets should be committed to rt-app. The current workload which is part of the PR has the purpose of showcasing that the mechanism works for complex workloads, it should not be part of the PR when it hypothetically gets merged.

Reply:
Mmm OK. I wonder what's the best way to make life easy for potential users of this new feature, though. I guess having a very simple example in the repo would certainly help. And I also guess adding substantial documentation to tutorial.txt should help as well.

Reply:
created a simple workload which just prints an integer. The workload has two methods that behave differently with respect to the integer increment.

Reply:
Added documentation to doc/tutorial.txt detailing the external_workload event.